Index: testi/Simulaattori.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package testi;\r\n\r\nimport simu.framework.Moottori;\r\nimport simu.framework.Trace;\r\nimport simu.framework.Trace.Level;\r\nimport simu.model.OmaMoottori;\r\n\r\npublic class Simulaattori { //Tekstipohjainen\r\n\r\n\tpublic static void main(String[] args) {\r\n\t\t\r\n\t\tTrace.setTraceLevel(Level.INFO);\r\n\t\tMoottori m = new OmaMoottori();\r\n\t\tm.setSimulointiaika(60);\r\n\t\tm.aja();\r\n\t\t///\r\n\t}\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/testi/Simulaattori.java b/testi/Simulaattori.java
--- a/testi/Simulaattori.java	(revision 85df5c2b715e06579547a23bc3cdfb9b8d46f13b)
+++ b/testi/Simulaattori.java	(date 1707395688126)
@@ -16,3 +16,4 @@
 		///
 	}
 }
+
